CHANGELOG.md

```diff
@@ -5,20 +5,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```

Comment:
nitpick(non-blocking): do you need this block before we have any unreleased changes?

Reply:
I like having it. It's a nice reminder for any future PRs or changes to accrue there. (We might not release a version after every change, necessarily).